app/src/main/java/io/intrepid/contest/screens/sendinvitations/SendInvitationsActivity.java

```diff
@@ -48,16 +49,20 @@ public static Intent makeIntent(Context context) {
     }
 
     @Override
+    protected int getLayoutResourceId() {
+        return R.layout.activity_send_invitations;
+    }
+
+    @Override
     protected void onViewCreated(Bundle savedInstanceState) {
         super.onViewCreated(savedInstanceState);
 
-        setActionBarTitle(R.string.invite_contestants_bar_title);
         setActionBarDisplayHomeAsUpEnabled(true);
     }
 
     @Override
-    protected int getLayoutResourceId() {
-        return R.layout.activity_send_invitations;
+    public void setActionBarTitle(@StringRes int titleResource) {
```

Comment:
ohhh I gotcha. Hmm that does seem a bit strange. but you have the @StringRes tag so I think its ok.